Cascade/Cascade-Bridging-Header.h

```diff
@@ -0,0 +1,5 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
+#import <PureLayout/PureLayout.h>
```

Comment:
You shouldn't do this. This have broken CI. You can use `PureLayout` from swift using ```import PureLayout```